# Custom Message to store the Wrench data
uint64 timestamp # Timestamp of the message

float32 f_x # Force in the inertial frame in x-axis
float32 f_y # Force in the inertial frame in y-axis
float32 f_z # Force in the inertial frame in z-axis

float32 tau_p # Torque in the body-fixed frame about b1-axis
float32 tau_q # Torque in the body-fixed frame about b2-axis
float32 tau_r # Torque in the body-fixed frame about b3-axis

